retail/interactive-tutorial/RetailProducts.Samples/UpdateProductSample.cs

```diff
@@ -16,6 +16,7 @@
 // Update product from a catalog using Retail API
 
 using Google.Cloud.Retail.V2;
+using Google.Protobuf.WellKnownTypes;
```

Comment:
I don't think you need this? What am I missing?

Reply:
we paste code during tutorial step, UpdateMask field in updateProductRequest variable